purview/utils/teiid.py

```diff
@@ -0,0 +1,87 @@
+# Taken from MARS project
+import logging
+
+import psycopg2
+
+LOG = logging.getLogger(__name__)
```

Comment:
Can we use a global `log` variable defined in `purview/__init__.py`? This way we don't configure logging per file.

Reply:
Isn't logging-per-file preferable? It gives you a better indication of where the log message is coming from, and finer-grained control over logging levels.